presto-iceberg/src/main/java/com/facebook/presto/iceberg/rest/IcebergRestConfig.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.rest;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.Optional;
+
+public class IcebergRestConfig
```

Comment:
Should we move this change to the first commit?

Reply:
I'm seeing this in the first commit currently. I guess we can both double check once I push my latest updates in a few moments 🙂

Reply:
Yes, you are right. I looked at an older version :-).